jr.py

```diff
@@ -59,6 +59,8 @@ def main(stdscr):
         cur_file = open(filename, "r")
         frames.append(parseFile(cur_file))
     frames.append(parseFile(open("required/jason.txt", "r")))
+    emptyFile = [[" "] * width] * height
+    frames.append(emptyFile)
```

Comment:
emptyFile isn't used, you can do 

`frames.append([[" "] * width] * height)`

to be more pythonic